tests/integration/targets/lookup_merge_variables/runme.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020, Thales Netherlands
+# Copyright (c) 2021, Ansible project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+set -eux
+
+ANSIBLE_ROLES_PATH=../ ANSIBLE_LOG_PATH=/tmp/ansible-test-merge-variables \
+    ansible-playbook test_merge_lookup.yml "$@"
```

Comment:
Why have a `runme.sh` if you don't do anything special (like installing Python dependencies)?

Reply:
Because I want to set the `log_path`, however, the integration tests don't seem to be picking up the `ansible.cfg` I initially created. With the `runme.sh` and using the environment variable that works (I had a look at how other integration tests where passing options, and the `runme.sh` seemed like the way to go).